src/Bicep.Core/Registry/AzureContainerRegistryManager.cs

```diff
@@ -68,46 +79,54 @@ async Task<OciArtifactResult> DownloadManifestInternalAsync(bool anonymousAccess
             string? mediaType,
```

Comment:
moved documentationUri/description concepts are better handled by the module-specific caller, so creating layers is now handled in module-specific code